eliot/symbolicate_resource.py

```diff
@@ -113,7 +113,7 @@ class InvalidStacks(Exception):
 
 # A valid debug filename consists of zero or more alpha-numeric characters, some
 # punctuation, and spaces.
-VALID_DEBUG_FILENAME = re.compile(r"^([A-Za-z0-9_.+{}@<> -]*)$")
+VALID_DEBUG_FILENAME = re.compile(r"^([A-Za-z0-9_.+{}@<> -~]*)$")
```

Comment:
I believe the hyphen `-` needs to be last (or first) so it isn't treated as a character range from space to tilde (i.e., every ASCII printable).

Reply:
Yep, just tested this and you're correct - good catch!